Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestManager.java

```diff
@@ -1607,40 +1579,10 @@ public void run() {
                     try {
                         processJobs();
                     } catch (Exception ex) { // Exception firewall
-                        if (jobProcessingTaskFuture.isCancelled()) {
-                            break;
-                        }
-                        if (ex instanceof CoordinationServiceException || ex instanceof AutoIngestJobNodeData.InvalidDataException) {
-                            errorState = ErrorState.COORDINATION_SERVICE_ERROR;
-                        } else if (ex instanceof SharedConfigurationException) {
-                            errorState = ErrorState.SHARED_CONFIGURATION_DOWNLOAD_ERROR;
-                        } else if (ex instanceof ServicesMonitorException) {
-                            errorState = ErrorState.SERVICES_MONITOR_COMMUNICATION_ERROR;
-                        } else if (ex instanceof DatabaseServerDownException) {
```

Comment:
@rcordovano Doesn't this completely remove auto-pause concept, including for core service outages (database, KWS)? I believe both Brian and sys admins (and me) wanted to keep auto-pausing if the services are down.

Reply:
Yes, it does. We can discuss why I did this in Jira or in person.